src/pages/edge/adobe-journey-optimizer-decisioning/api-reference.md

```diff
@@ -55,7 +55,11 @@ iOS
 
 ## onPropositionsUpdate
 
-This API registers a permanent callback which is invoked whenever the Edge extension dispatches a response Event received from the Experience Edge Network upon a personalization query. The personalization query requests can be triggered by the `updatePropositions` API, Edge extension `sendEvent` API or launch consequence rules.
+This API registers a permanent callback which is invoked whenever the Edge extension dispatches a response event received from the Experience Edge Network upon a personalization query. Additionally, the callback is only invoked if the response event payload has at least one proposition with valid, non-empty offers. The personalization query requests can be triggered by the `updatePropositions` API.
```

Comment:
> This API registers a permanent callback which is invoked whenever the Edge extension dispatches a response event received from the Experience Edge Network upon a personalization query

isn't this always an `eventType` of `personalization.response`?  maybe it would be easier to just name it rather than saying "upon a personalization query"?

something like 
 
> This API registers a permanent callback which is invoked whenever the Edge extension processes a `personalization.response` event from the Experience Edge Network.

Reply:
rewording option

> if the response event payload has at least one proposition with valid, non-empty offers.
 
> if the response event contains at least one valid offer.

Reply:
makes sense, updated